lib/fbe/iterate.rb

```
@@ -155,6 +156,10 @@ def by(query)
   def as(label)
     raise 'Label is already set' unless @label.nil?
     raise 'Cannot set "label" to nil' if label.nil?
+    if label.include?('-')
+      @loog.warn('Use snake_case instead of kebab-case for label in Fbe::Iterate, ' \
```

@Yegorov let's just raise an exception, no need to be soft on this